### PR TITLE
Adding the terms 'Numpy' and 'SciPy' in en, pt and es, and translate Python from Portuguese and Spanish

### PR DESCRIPTION
**Pull request by leticiadasilva**
…se, and adding the translate of the definition of 'Python' in portuguese and spanish.

## Author: 
@leticiadasilva

## Language:  

Portuguese

## Terms defined:
- numpy
- scipy
- python

## Editor

Assign the PR based on the language to the following person:

@beatrizmilz 
@zkamvar
@ian-flores

### CODE REVIEW

**Comment by leticiadasilva** [comment]
If you can adding the label "hacktoberfest-accepted" after the PR will be accepted, I thank you!

**Review by zkamvar — CHANGES_REQUESTED** [review]
Looks good with minor syntax and spelling changes for English. 

IMO, I don't think we should start the definitions with "Is/Es/É" for the sake of clarity

**Comment by leticiadasilva** [comment]
@zkamvar thank you for the suggestions! :smiley:

**Review by zkamvar — APPROVED** [review]

**Comment by zkamvar** [comment]
@ian-flores and @beatrizmilz, do these changes look good?

File: glossary.yml [hunk 1]
```diff
@@ -2990,6 +2990,30 @@
...
+    term: "numpy"
+    def: >
+      Es un paquete Python de código abierto que le permite trabajar con arreglos, vectores y matrices de 
+      dimensión N, en un método comparable y con una sintaxis similar al software Matlab. Puede encontrar 
+      funciones y operaciones sofisticadas, enfocadas en matrices multidimensionales, álgebra lineal, 
+      transformada de Fourrier y generación de valores aleatorios.
+  pt:
+    term: "numpy"
+    def: >
+      É um pacote Python open source que permite trabalhar com arrays, vetores e matrizes de dimensão N,
```
Comment (beatrizmilz) [inline glossary.yml:3013]:
```suggestion
      É um pacote Python de código aberto que permite trabalhar com arrays, vetores e matrizes de dimensão N, 
```

File: glossary.yml [hunk 2]
```diff
@@ -3128,6 +3152,29 @@
...
+    def: >
+      Es un paquete de Python de código abierto que ofrece estructuras de datos rápidas, 
+      flexibles y expresivas para que trabajar con series de tiempo estructuradas sea fácil 
+      e intuitivo. Se utiliza como una poderosa herramienta para el análisis y la manipulación 
+      de datos.
+  pt:
+    term: "pandas"
+    def: >
+      É um pacote Python de código aberto que oferece estruturas de dados rápidas, flexíveis
+      e expressivas para tornar o trabalho com séries temporais estruturadas fácil e
```
Comment (beatrizmilz) [inline glossary.yml:3175]:
```suggestion
      É um pacote Python de código aberto que oferece estruturas de dados rápidas, flexíveis
      e expressivas para tornar o trabalho com dados estruturados e séries temporais fácil e 
```

File: glossary.yml [hunk 3]
```diff
@@ -3459,6 +3506,16 @@
...
+  es:
+    term: "Python"
+    def: >
+      Un lenguaje de programación de código abierto interpretado popular que 
+      se basa en la sangría para definir la estructura de control.
+  pt:
+    term: "Python"
+    def: >
+      Uma linguagem de programação de código aberto interpretada popular que 
+      depende de indentação para definir a estrutura de controle.
```
Comment (beatrizmilz) [inline glossary.yml:3518]:
```suggestion
      Uma popular linguagem de programação interpretada, de código aberto,  que 
      depende de indentação para definir a estrutura de controle.
```

**Review by beatrizmilz — COMMENTED** [review]
Oi @leticiadasilva ! Obrigada por suas contribuições e desculpa a demora para revisar. Fiz algumas sugestões buscando deixar padronizado (por exemplo _Open Source_ sugeri mudar para _Código Aberto_), e outras pois parecia que ajudava na fluência da leitura da frase. O que acha das sugestões?

File: glossary.yml [hunk 4]
```diff
@@ -2990,6 +2990,30 @@
     def: >
       An "expression" with no arguments, such as the value 3.
 
+- slug: numpy
+  ref:
+    - python
+  en:
+    term: "numpy"
+    def: >
+      An open source Python package  that allow work with arrays, vectors and matrices of dimension N,
```
[the remark this answers is not in the document]
Comment (zkamvar) [reply glossary.yml:2999]: @beatrizmilz, revised. Thank you for catching that!

**Comment by zkamvar** [comment]
I'll go ahead and label this with hacktoberfest-accepted so that you can receive credit for it even though the review on this is taking a bit longer than expected.

**Comment by leticiadasilva** [comment]
@beatrizmilz muito obrigada pelas sugestões, eu adorei! :heart:

**Comment by leticiadasilva** [comment]
@zkamvar Thank you!!! :smile:

**Comment by beatrizmilz** [comment]
For me is all great with the translation to Portuguese! @ian-flores Can you check the translations to Spanish, please? Thank you all

**Review by ian-flores — APPROVED** [review]

**Comment by elletjies** [comment]
@all-contributors please add @leticiadasilva for translation

**Pull request merged**